napari/utils/migrations.py

```diff
@@ -61,3 +62,49 @@ def _update_from_dict(*args, **kwargs):
         return _update_from_dict
 
     return _wrapper
+
+
+def add_deprecated_property(
+    obj: Any,
+    previous_name: str,
+    new_name: str,
+    version: str,
+    since_version: str,
+) -> None:
+    """
+    Adds deprecated property and links to new property name setter and getter.
```

Comment:
I prefer to see something highlighting that this is only for renamed properties. But maybe it is only my language worries.

Reply:
Done